CHANGELOG.md

```diff
@@ -25,7 +26,7 @@ _None_
 ### New Features
 
 * Support for a `version.properties` to manage app versioning - all existing paths remain intact and new paths are only used when a `version.properties` file is present.
-* Add support for providing an `app:` parameter to most versioning-related actions to allow support for multiple apps hosted in a monorepo
+* Add support for providing an `app:` parameter to most versioning-related actions to allow support for multiple apps hosted in a monorepo.
```

Comment:
I'm still not convinced we need / should maintain this given the fact that we're using a common version between apps sharing a repo right now. I feel like this will be a future maintenance burden for something we _might_ need in the future. WDYT?

Reply:
I agree, but this change was already shipped in 1.3.0 (this entry is from `## 1.3.0`, the only change I made on this line is to fix a missing `.`)

My take is that we should at least avoid having the toolkit in its current version crash, so still accept and land this `1.3.1` hotfix as is, to at least get back to a non-crashing version. And only after that lands, revert the changes as we discussed (to get back to using a common version for both WP and JP and remove that `app:` parameter) for a future 1.4.0 – well actually a 2.0.0 as this would be a breaking change I think.